src/App.js

```diff
@@ -40,12 +58,27 @@ class Container extends React.Component {
   }
 
   render() {
+    var layout = [
+      {i: 'a', x: 0, y: 0, w: 3, h: 1, minW: 2, maxW: 4}
+    ];
+
     return (
       <div>
-        <Line percent={this.state.progress} />
-        <button onClick={this.playSound}>
-          Sound 1
-        </button>
+        <div className='header'>
+          <Line className='progressBar' percent={this.state.progress} />
+          <div className='editModeBar'>
+            <h1>Edit Mode: </h1>
+            <ToggleButtonGroup name="editMode" type="radio" onChange={this.changeEditMode}>
```

Comment:
Edit Mode 'OFF' button should be pre-checked. You can do this by adding `defaultValue={false}` to the ToggleButtonGroup.

Reply:
Fixed